interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -520,6 +520,22 @@ private void runTest(TestCases testCase) throws Exception {
         tester.performSoakTest(
             serverHost,
             false /* resetChannelPerIteration */,
+            false /* concurrent */,
+            soakIterations,
+            soakMaxFailures,
+            soakPerIterationMaxAcceptableLatencyMs,
+            soakMinTimeMsBetweenRpcs,
+            soakOverallTimeoutSeconds,
+            soakRequestSize,
+            soakResponseSize);
+        break;
+      }
+
+      case RPC_SOAK_CONCURRENT: {
```

Comment:
Rather than introduce new test cases, let's reuse the existing `rpc_soak` and `channel_soak` test cases, but add one more parameter for number of threads (passing the value in from the aforementioned `--soak_num_threads` flag)